buster/documents_manager/base.py

```diff
@@ -58,7 +57,8 @@ def add(
         self,
         df: pd.DataFrame,
         num_workers: int = 16,
-        embedding_fn: callable = get_openai_embedding,
+        embedding_fn: Callable[[str], np.ndarray] = get_openai_embedding,
+        sparse_embedding_fn: Callable[[str], dict[str, list[float]]] = None,
```

Comment:
can you add docstrings for what sparse_embeddinf_fn does and when it shoudl be used?